### DISCUSSION TITLE
Github Classroom: Internal server error 500 when accepting assignment

### DISCUSSION THREAD

**Discussion by saulpink** [community/community#141812] [GitHub Education]
### Hi everyone,

In the class I TA for, every time I try to accept an assignment using an invite link, I get this Internal Service Error 500. I've tried every browser, multiple networks, clearing cookies and cache, logging in and out, removing myself and rejoining the classroom, etc. Nothing seems to work. 

I've also checked out other threads of folks who've had the same/similar problems, and their workarounds do not work either.

<img width="908" alt="Screenshot 2024-10-17 at 12 05 23 PM" src="https://github.com/user-attachments/assets/6a4c315c-fbf2-4460-9d19-b066253c0e1f">

**Comment by jonjoncardoso** [comment]
I am having the exact same issue. Very frustrating

**Comment by osx35** [comment]
I am also affected by this error. Makes every class harder and my teacher does not see my activity.
